sites/partners/src/pages/listings/[id]/applications/pending/index.tsx

```diff
@@ -51,27 +52,23 @@ const ApplicationsList = () => {
       cellRenderer: "formatLinkCell",
       valueGetter: ({ data }) => {
         if (!data?.applications?.length) return ""
-        const applicant = data.applications[0]?.applicant
 
-        return `${applicant.firstName} ${applicant.lastName}: ${data.rule}`
+        const applications = data.applications.reduce((acc, curr) => {
```

Comment:
I don't think we need to address this here/in this pr but I'm curious on your thoughts on pulling stuff like this out into helper files so they are more easily unit testable

Reply:
did both! 🫡